docs/griptape-framework/misc/events.md

```diff
@@ -74,6 +74,18 @@ Handler 1 <class 'griptape.events.finish_structure_run_event.FinishStructureRunE
 Handler 2 <class 'griptape.events.finish_structure_run_event.FinishStructureRunEvent'>
 ```
 
+## Stream Iterator
+
+You can use `Structure.run_stream()` for streaming Events from the `Structure` in the form of an iterator.
```

Comment:
Would prefer a `EventStream` util like `Stream` that currently exists and punt more core API changes to the Structure api to a later time, but not going to block the PR based on that.

Reply:
The downside of this approach is that `EventStream` would need to support anything that might need this functionality (Structures, Tasks, Drivers, Tools, etc).

I'm just as nervous about modifying `Structure`'s API, but this implementation does at least seem consistent with other similar frameworks.